scripts/workers/auto_start_workers.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
Can we please change the name of file to more meaningful as the script does both starting and stopping of workers.